learning_paths/migrations/0007_replace_uuid_with_learningpathkey.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 4.2.16 on 2025-03-12 13:08
+
+from django.db import migrations, models
+import learning_paths.keys
+import uuid
+
+
+def set_keys_from_uuids(apps, schema_editor):
+    """
+    Assign unique keys to existing learning paths based on their UUIDs.
+    """
+    LearningPath = apps.get_model("learning_paths", "LearningPath")
+
+    for learning_path in LearningPath.objects.all():
+        key_str = f"path-v1:test+test+test+{learning_path.uuid}"
```

Comment:
This plugin has not been deployed anywhere yet, so this function will only affect dev environments.